test/e2e/benchmark/manifest.go

```diff
@@ -95,3 +96,21 @@ func (m *Manifest) GetConsensusParams() *tmproto.ConsensusParams {
 	cparams.Block.MaxBytes = m.MaxBlockBytes
 	return cparams
 }
+
+// summary generates a summary of the Manifest struct to be used as chain id.
+func (m *Manifest) summary() string {
+	latency := 0
+	if m.EnableLatency {
+		latency = 1
+	}
+	maxBlockMB := m.MaxBlockBytes / mb
```

Comment:
are `mb` and `MB` duplicates? https://github.com/celestiaorg/celestia-app/blob/77042f80d51b32afac48201a4bcf956976a14f8f/test/e2e/testnet/defaults.go#L12

I didn't look, but I'm assuming its some import cycle thing, if so is it possible to remove that in a different PR?

Reply:
Not actually, there is no import cycle, it is now using the testnet's constants https://github.com/celestiaorg/celestia-app/pull/3612/commits/d93c2aad8914c58324936b4e2f8698ecd618e906